test/testinput/bufr_ncep_prepbufr_adpupa.py

```diff
@@ -67,10 +68,19 @@ def test_bufr_to_ioda(DATA_PATH, OUTPUT_PATH, date):
     pob *= 100
 
     # Time variable
+    # The time is entering in as a float32 value representing an offset from the cycle
+    # time in hours (so fractions of hours can exist). This needs to be first converted
+    # to seconds while still a float32, then converted to an int64 for the dateTime variable.
+    # Another consideration is that the get function returns a masked array with an
+    # appropriate fill value assigned. When converting to an int64, the fill value needs
+    # to get updated, and then before writing into the output ioda file, the masked array
+    # function filled() needs to be called which will convert the values marked invalid
+    # to the fill value.
     hrdr = r.get('timeOffset')
```

Comment:
delete this line

Reply:
Thanks for catching this. I've removed the line.